### PR TITLE
Feat/header

### PR DESCRIPTION
**Pull request by aiskairr**
Issue #:
[Notion](https://trello.com/c/mVsTwz08/10-header-adaptive-layout)

## PR type

Please check the type of change your PR introduces:

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### before:
before

#### after:
after
adaptive1
adaptive2
ьщвфд

## Does this introduce a breaking change?

- [x] Yes (please describe):
- [ ] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [x] Responsive layout and functionality have been tested
  - [x] desktop and mobile view responsiveness are supported
  - [x] no loss of functionality or content when zoomed in to 400%
  - [x] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

**Review by Adilxa — APPROVED** [review]
Gj

**Comment by asylbekovkairat** [comment]
тут грамматическая ошибка "Зарегистрироваться"*

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+              className={scss.burger__menu}
+            >
+              <img src="./images/header/BurgerMenu.svg" alt="BurgerMenuIcon" />
+            </button>
+            <div className={scss.left__content}>
+              <Link to="/">
+                <img src="./images/header/Logo.svg" alt="LogoImage" />
+              </Link>
+              <nav>
+                <Link to="#">{t("header.Perfomers")}</Link>
```
Comment (asylbekovkairat) [inline src/components/header/Header.jsx:26]: нужны нормальные роуты для линк

File: src/components/header/Header.module.scss [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        img{
+          margin-bottom: 10px;
+        }
+      }
+      nav{
+        width: 75%;
+        display: flex;
+        justify-content: space-between;
+        a {
+          font-family: "Montserrat", sans-serif;
```
Comment (asylbekovkairat) [inline src/components/header/Header.module.scss:39]: сделай этот стиль глобальным

File: src/components/header/Header.module.scss [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+      }
+    }
+    .right__content {
+      height: 100%;
+      width: 45%;
+      display: flex;
+      justify-content: flex-end;
+      align-items: center;
+      a:first-child {
+        font-family: "Montserrat", sans-serif;
```
Comment (asylbekovkairat) [inline src/components/header/Header.module.scss:58]: переносим все стили которые много раз используются в глобальный файл

File: src/components/header/Header.module.scss [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+        font-style: normal;
+        font-weight: 600;
+        font-size: 13.2423px;
+        line-height: 16px;
+        text-decoration: #3a204f;
+        color: #3a204f;
```
Comment (asylbekovkairat) [inline src/components/header/Header.module.scss:64]: где миксины???

**Pull request merged**